rero_ils/modules/items/api.py

```diff
@@ -602,10 +602,16 @@ def extend_loan(self, current_loan, **kwargs):
 
     @add_loans_parameters_and_flush_indexes
     def request(self, current_loan, **kwargs):
-        """Request item for the user."""
+        """Request item for the user and create notifications."""
         loan = current_circulation.circulation.trigger(
             current_loan, **dict(kwargs, trigger='request')
         )
+
+        checked_out_loan_pid = self.get_loan_pid_with_item_on_loan(self.pid)
```

Comment:
this can be done in the loan itself, by the trigger

Reply:
ok